app/forms/course_details_form.rb

```diff
@@ -72,6 +78,7 @@ def other_age_range?
 
   def update_trainee_attributes
     trainee.assign_attributes({
+      course_code: nil,
```

Comment:
is this always supposed to be nil?

Reply:
this resets the course code value if a course has been entered manually via this form